src/yunohost/utils/packages.py

```diff
@@ -21,15 +21,12 @@
 import re
 import os
 import logging
-from collections import OrderedDict
 
-import apt
-from apt_pkg import version_compare
```

Comment:
aaaaannnnnd right after pressing the button I realize `version_compare` is still used in the damn Specifier / SpecifierSet which is also completly overkilled (iirc we only use it to check the version requirement in app manifest ... and it's always something like `"< x.y.z"` and we could just do this using any reasonable lightweight lib or whatever ... like the one Josue uses in his PR ...